aptos-move/framework/aptos-framework/sources/stake.move

```diff
@@ -1643,7 +1603,7 @@ module aptos_framework::stake {
     }
 
     #[test(aptos_framework = @0x1, core_resources = @core_resources, validator = @0x123)]
-    #[expected_failure(abort_code = 0x1000f)]
+    #[expected_failure(abort_code = 0x1000E)]
```

Comment:
why this change?

Reply:
The's because I removed one error code and shifted others. After code freeze we should never do that.